skunkbooth/filters/cold.py

```diff
@@ -0,0 +1,29 @@
+from PIL import Image
+from PIL.ImageOps import grayscale, colorize
+
+
+class filter:
+    """Cold filter."""
+
+    def __init__(self):
+        """
+        Init required values.
+
+        textOp True = operation on ASCII art, False = operation on PIL image
+        name will be the name of the filter to be used for display/indexing
+        """
+        self.textOp = False
+        self.name = "Cold Filter"
```

Comment:
I'd just call this filter 'Cold' to keep filter naming consistent with 'Warm'

Reply:
Will do